src/Resources/translations/validators.es.xliff

```diff
@@ -12,6 +12,10 @@
         <target>La URL raíz '/' ya está asociada a otra página de este sitio, por favor selecciona un padre.
                 </target>
       </trans-unit>
+      <trans-unit id="error.uniq_url.no_site" resname="error.uniq_url.no_site">
+        <source>error.uniq_url.no_site</source>
+        <target>error.uniq_url.no_site</target>
```

Comment:
Please dont add this copy unless you have the translation for it.

Reply:
In the other files, the other translations do it too

Reply:
We try to stop this because it's not a good practice.

With a placeholder, you get the placeholder
Without the translation, it default to the english one, which is better